cards.js

```diff
@@ -17,7 +17,57 @@ function shuffle(array){
 }
 
 function buildDeck(){
+  var deck = []
 
-// Put some code in here!
+  var suit = ""
+  //var face_value = "2"
 
+  for (value_index in values){
+    for(card_suit in suits){
+      var card = {}
+
+
+      card.suit = suits[card_suit];
+      card.face_value = values[value_index];
+      // console.log(card)
+      deck.push(card);
```

Comment:
If we also have an attribute for the value_index of the card, we could use that later to compare two cards.